### DISCUSSION TITLE
Any beautifier I can use for liquid templates?

### DISCUSSION THREAD

**Discussion by denik1981** [harttle/liquidjs#421] [Ideas]
I'm currently using VSCODE with unibeautfier that provides a JS-Beautify implementation to format liquid templates. Works well but it is far from perfect. Neverless, I haven't found a good plugin for Prettier either. What are you using or where should I look in order to get my liquid code nicely formatted?

**Comment by harttle** [comment]
Recently I also had some conversations offline for this, maybe I'll figure out how to publish a separate parser. Then we can:

- parse to AST tree and used by other tools
- create a beautifier, a syntax highlighter, AST viewer, etc.
